robyn/types.py

```diff
@@ -18,5 +20,22 @@ def as_list(self):
         ]
 
 
+PathParams = NewType("PathParams", Dict[str, str])
+RequestMethod = NewType("RequestMethod", str)
+RequestURL = NewType("RequestURL", Url)
+FormData = NewType("FormData", Dict[str, str])
+RequestFiles = NewType("RequestFiles", Dict[str, bytes])
+RequestIP = NewType("RequestIP", Optional[str])
+RequestIdentity = NewType("RequestIdentity", Optional[Identity])
+
+
 class JSONResponse(TypedDict):
     pass
+
+
+class RequestBody:
+    pass
+
+
+class QueryParam:
+    pass
```

Comment:
```suggestion
class QueryParams:
    pass
```

@VishnuSanal , this should be QueryParams